assets/css/components.css

```diff
@@ -171,4 +171,13 @@
   .info-box {
     @apply p-4 bg-gray-100 text-sm text-gray-500 font-medium rounded-lg;
   }
+
+  /* Tab Icons */
+  .hover-green:hover .icon {
+    @apply text-green-500 font-medium;
+  }
+
+  .hover-purple:hover .icon {
+    @apply text-purple-500 font-medium;
+  }
```

Comment:
With tailwind we generally try to minimize the amount of custom classes, unless they are kinda necessary. In this case we can totally just use these inline :)

```suggestion
```